Modules/HR/Config/config.php

```diff
@@ -230,4 +230,8 @@
         'acc_number' => 'Account Number',
         'ifsc_code' => 'IFSC Code',
     ],
+    'follow-up-attempts' => '2',
+    'follow_up_recipients' => [
+        'primary' => env('HR_NEED_FOLLOW_UP_REMINDER_EMAIL', 'ayush.uniyal@coloredcow.in'),
```

Comment:
Here we need to change this email, I will let you know, once confirmed.
```suggestion
        'primary' => env('HR_NEED_FOLLOW_UP_REMINDER_EMAIL', 'ayush.uniyal@coloredcow.in'),
```

Reply:
> Here we need to change this email, I will let you know, once confirmed.

@Ayush8923 ok